File: lab1/Variant51.py
import math
x = float(input("Введите x "))
a = float(input("Введите a "))
z = int(input("Введите номер функции "))
if z == 1:
    g = (3*(-3*a**2+2*a*x+21*x**2))/(35*a**2+37*a*x+6*x**2)
    print("g= %.3f" % g)
elif z == 2:
    f = 1/(math.cos(3*a**2+5*a*x+2*x**2))
    print("f= %.3f" % f)
elif z == 3:
    y = math.acosh(-12*a**2-4*a*x+x**2+1)
    print("y= %.3f" % y)

File: c/lab 1/Variant51.c
#include <stdio.h>
#include <math.h>

double acosh(double d);

int main()
{
    double a, x, g, f, y ;

    printf ( "Введите x и a \n" );
    scanf ( "%lf", &x );
    scanf ( "%lf", &a );
    g = (3*(-3*pow(a,2)+2*a*x+21*pow(x,2)))/(35*pow(a,2)+37*a*x+6*pow(x,2));
    f = 1/(cos(3*pow(a,2)+5*a*x+2*pow(x,2)));
    y = acosh(-12*pow(a,2)-4*a*x+pow(x,2)+1);
    printf ("g= %f \n", g) ;
    printf ("f= %f \n", f) ;
    printf ("y= %f \n", y) ;

    return 0;

}